delfin/db/sqlalchemy/models.py

```diff
@@ -366,7 +366,7 @@ class StorageHostGrpHostRel(BASE, DelfinBase):
     """Represents the storage host group and storage host relation
     attributes.
     """
-    __tablename__ = 'storage_host_grp_host_rels'
+    __tablename__ = 'storage_host_group_relations'
```

Comment:
Currently __tablename__ is in sync with class name, I think we can follow same convention here as well

Reply:
Done, thanks.